src/framework/components/element/component.js

```diff
@@ -465,19 +465,11 @@ class ElementComponent extends Component {
     }
 
     _bindScreen(screen) {
-        screen.on('set:resolution', this._onScreenResize, this);
-        screen.on('set:referenceresolution', this._onScreenResize, this);
-        screen.on('set:scaleblend', this._onScreenResize, this);
-        screen.on('set:screenspace', this._onScreenSpaceChange, this);
-        screen.on('remove', this._onScreenRemove, this);
+        screen._bindElement(this);
```

Comment:
might be worth adding a comment here explaining the optimisation.

Reply:
Done